### PR TITLE
fix frontend lint issues

### PR DESCRIPTION
**Pull request by erinz2020**
1. remove some unused variables
2. add some eslint-disable lines to disable eslint in some js files that uses node.js variables

does not fix a numbered issue

### CODE REVIEW

File: frontend/src/UnAuthenticatedSwitch.jsx [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
 import Unauthorized from "./pages/errorPages/Unauthorized";
-import About from "./About";
-import EncounterSearch from "./pages/EncounterSearch";
-import Home from "./pages/Home";
 
 export default function UnAuthenticatedSwitch({ showAlert, setShowAlert }) {
   const [header, setHeader] = React.useState(true);
-  const [headerTop, setHeaderTop] = React.useState("60px");
-  const alertBannerRef = React.useRef(null);
+  // const [headerTop, setHeaderTop] = React.useState("60px");
```
Comment (TanyaStere42) [inline frontend/src/UnAuthenticatedSwitch.jsx:12]: can these be removed since they're commented out?

File: frontend/src/components/Chip.jsx [hunk 2]
```diff
@@ -1,127 +1,137 @@
...
-        } else if (query?.biologicalMeasurements) {
-            Object.entries(query).forEach(([key, value]) => {
-                entries.push(`${filterKey || key} filter is set`);
-            });
-        }
+    if (query?.geo_bounding_box) {
+      const { top_left, bottom_right } =
+        query.geo_bounding_box["locationGeoPoint"];
+      entries.push(
+        `Location within bounding box: Northeast latitude: ${top_left.lat}, longitude: ${top_left.lon}, southwest latitude: ${bottom_right.lat}, longitude: ${bottom_right.lon}`,
```
Comment (TanyaStere42) [inline frontend/src/components/Chip.jsx:61]: extra `,` at the end I think

File: frontend/src/UnAuthenticatedSwitch.jsx (hunk 1 above)
Comment (erinz2020) [reply frontend/src/UnAuthenticatedSwitch.jsx]: deleted

**Review by TanyaStere42 — APPROVED** [review]
QA and general review found to be successful

**Pull request merged**